src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DisplayOutPut from './Display';
+import Buttons from './Buttons';
+import '../style/calculator.css';
+
+class Calculator extends React.Component {
```

Comment:
- [ ] Please use the `functional` component now you are using the `class` component so kindly replace this class component with the functional component :slightly_smiling_face: